types/vfile-rename-tests.ts

```diff
@@ -0,0 +1,20 @@
+import vfile = require('vfile')
+import rename = require('vfile-rename')
+
+const file = vfile('index.js')
+
+function move(file: vfile.VFile) {
+  file.stem = 'main'
+  return file
+}
+
+rename()
```

Comment:
:raised_eyebrow: nothing can be renamed? I'm not sure this makes sense.

Reply:
It could also be good to get a failing case in.
Something like

```ts
rename(1) // $ExpectError
```

Reply:
Technically vfile accepts undefined, so yes, I guess it works here. I can see the types being a bit stricter tho

Reply:
> 🤨 nothing can be renamed? I'm not sure this makes sense.

Yeah, I was thinking that was odd as well, however:

```
t.equal(String(rename()), '', 'should create a vfile (#1)')
```

From `test.js`.

Reply:
> Technically vfile accepts undefined, so yes, I guess it works here. I can see the types being a bit stricter tho

How much stricter would you like it?

Reply:
I’m fine with it being loose like this, although it’s not the typical case, so I’ll leave this up to @ChristianMurphy

Reply:
I'm fine with how it is now.
I believe the `?` optional, also adds `undefined` to the type as well, which covers that case.